lighthouse-cli/test/smokehouse/run-all-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+for d in "$DIR"/*/ ; do
+  bash "${d}run-tests.sh"
```

Comment:
there are a few other ways to do this, but the benefit of this approach is the failures won't be silent. :)

sgtm

Reply:
can we just manually list the files in here? If I'm messing with some smoke tests it's a lot easier to comment out lines in one place by name than look up where the individual run-tests file is

also it'd probably be better to just get rid of all the `run-tests.sh` files since they're all exactly the same except for the expectations and config filename strings :)

Reply:
will nuke 'em all in a follow up PR and improve this file here 👍